samples/test-samples.cmake

```diff
@@ -50,7 +50,7 @@ foreach (SAMPLE data-sealing file-encryptor helloworld local_attestation remote_
 
     # Build with pkg-config
     execute_process(
-      COMMAND sh -c "PATH=${INSTALL_DIR}/bin:$PATH PKG_CONFIG_PATH=${BUILD_DIR}/pkgconfig OE_PREFIX=${INSTALL_DIR} make -C ${SAMPLE_SOURCE_DIR} clean build run"
+      COMMAND ${CMAKE_COMMAND} -E env PATH=${INSTALL_DIR}/bin:$ENV{PATH} PKG_CONFIG_PATH=${BUILD_DIR}/pkgconfig OE_PREFIX=${INSTALL_DIR} COMMAND make -C ${SAMPLE_SOURCE_DIR} clean build run
```

Comment:
execute_process `COMMAND` is executed in [parallel](https://cmake.org/cmake/help/v3.12/command/execute_process.html) we could use add_custom_command [instead](https://cmake.org/cmake/help/v3.12/command/add_custom_command.html#command:add_custom_command)

Reply:
That's only if `COMMAND COMMAND COMMAND` is used; we're only using one command for each execute_process so this doesn't affect us.

...

And that's a typo on my part, that COMMAND doesn't belong, I was blindly following the man page for CMake in command line mode. Thanks!